packages/qwik/src/optimizer/src/plugins/plugin.ts

```diff
@@ -895,18 +892,6 @@ export const manifest = ${JSON.stringify(manifest)};\n`;
         }
       }
     }
-
-    if (id.includes('node_modules')) {
-      return null;
-    }
-
-    // Patch to prevent over-prefetching, we must clearly separate .tsx/.jsx chunks so that rollup doesn't mix random imports into non-entry files such as hooks.
-    // Maybe a better solution would be to mark those files as entires earlier in the chain so that we can remove this check and the one above altogether.
-    // We check .(tsx|jsx) after node_modules in case some node_modules end with .jsx or .tsx.
-    if (/\.(tsx|jsx)$/.test(id)) {
```

Comment:
removed this workaround as does not seem to be needed with insights. If insights is not used then rollup does a good job at bundling.